apps/src/music/blockly/MusicBlocklyWorkspace.ts

```diff
@@ -88,6 +88,20 @@ export default class MusicBlocklyWorkspace {
 
     const toolboxBlocks = getToolbox(getBlockMode(), toolbox);
 
+    // Other Blockly labs use FeedbackUtils.prototype.showSimpleDialog to create a prettier dialog.
+    // See StudioApp.prototype.inject for more information.
```

Comment:
Maybe worth a note that this is temporarily only used in the advanced model, and only for renaming variables?

Reply:
Can do!

Reply:
Added comment. PTAL